feature-libs/organization/order-approval/components/list/order-approval-list.component.html

```diff
@@ -1,6 +1,11 @@
 <ng-container *ngIf="orderApprovals$ | async as orderApprovals">
   <ng-container
-    *ngIf="orderApprovals.pagination.totalResults > 0; else noOrder"
+    *ngIf="
+      orderApprovals.pagination &&
+        orderApprovals.pagination.totalResults !== undefined &&
```

Comment:
Could we just check for a truthy value instead of specifically `!== undefined` ?